go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/spf13/afero
 
 require (
-	github.com/pkg/sftp v1.10.1
-	golang.org/x/crypto v0.0.0-20190820162420-60c769a6c586
-	golang.org/x/text v0.3.3
+	github.com/pkg/sftp v1.13.1
```

Comment:
This also fixes a security vulnerability marked by Snyk, which has been fixed in v1.11.0.